pkg/kubelet/kubelet.go

```diff
@@ -1910,6 +1910,10 @@ func (kl *Kubelet) handleMirrorPod(mirrorPod *v1.Pod, start time.Time) {
 func (kl *Kubelet) HandlePodAdditions(pods []*v1.Pod) {
 	start := kl.clock.Now()
 	sort.Sort(sliceutils.PodsByCreationTime(pods))
+
+	// Pass critical pods through admission check first.
+	sort.Sort(kubepod.SortPrioritizingCriticalPods(pods))
```

Comment:
Do we still want the critical/non-critical pods to be sub-sorted by their creation time? I don't think `sort.Sort` guarantees a stable sort.